lib/worker.js

```diff
@@ -13,6 +13,12 @@ module.exports = async function ({ targets }) {
   targets = await Promise.all(targets.map(async (t) => {
     let fn
     try {
+      if(typeof process !== "undefined"){
+        //reuse the error code.
+        const err = new Error()
```

Comment:
```suggestion
        const err = Error()
```

Also, an appropriate message should be passed to the constructor.

Reply:
We should not emit Node core error codes